.readthedocs.yml

```
@@ -0,0 +1,23 @@
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  builder: dirhtml
+  configuration: docs/conf.py
+  fail_on_warning: true
```

Let's temporarily disable this:
```suggestion
  # fail_on_warning: true
```